Yona/Yona/Welcome/WelcomeViewController.swift

```diff
@@ -118,4 +81,32 @@ class WelcomeViewController: BaseViewController {
         }
     }
     
+    fileprivate func processEnvironmentURL(newEnvironmentURL:String, currentEnvironmentURL:String) {
+        EnvironmentManager.updateEnvironment(withURLString: newEnvironmentURL)
+        self.didEditTextField = false
+        if currentEnvironmentURL != newEnvironmentURL{
```

Comment:
See my earlier comment about the code formatter. This line misses the parenthesis around the condition of the if-statement.

Reply:
As you said I've modified that.
I followed Swift Coding Standard from here: https://github.com/raywenderlich/swift-style-guide#parentheses
For future, We can stick to any one of these coding style. Please let me know.

Reply:
It looks awkward to me, but that's taste. I didn't know it's common to omit these with Swift. The real issue is consistence. Several of your new if-statements have parentheses, but this one didn't have them.

I now see the general convention in our code is to omit them, so please consistently omit them on this change set.